README.md

```diff
@@ -1,5 +1,23 @@
 # OpenT2T
-Cross-platform core libraries for Open Translators to Things
+Cross-platform core libraries for Open Translators to Things written in TypeScript
+
+# Compiling the .ts to .js
+
+- npm run build
+
+# Running unit tests
+- npm run test
+
+# Running the linter
+- npm run lint
+
+# Preparing the package for publishing
+1. Update version number in package.json; Send a PR out for approval/FYI
+2. Follow steps to have the changes merged to Master
+3. cd to the node folder and run the following:
+4. npm run build
+5. npm run prepublish
+6. npm publish
```

Comment:
Doesn't the prepublish script get run automatically by `npm publish`? And that would also include build. So you can skip steps 4 and 5.